Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -643,12 +664,445 @@ def resume_scan_command():
+    vpr_score = {}
+    if vpr_score_value:
+        vpr_score = {'eq': [float(x) for x in argToList(vpr_score_value)] if args.get('vprScoreOperator') == 'equal' else None,
+                     'neq': [float(x) for x in argToList(vpr_score_value)]
+                     if args.get('vprScoreOperator') == 'not equal' else None,
+                     'gt': float(vpr_score_value) if args.get('vprScoreOperator') == 'gt' else None,
+                     'lt': float(vpr_score_value) if args.get('vprScoreOperator') == 'lt' else None,
+                     'gte': float(vpr_score_value) if args.get('vprScoreOperator') == 'gte' else None,
+                     'lte': float(vpr_score_value) if args.get('vprScoreOperator') == 'lte' else None}
+
+    if args.get('vprScoreRange'):
+        lower_range_bound, upper_range_bound = validate_range(args.get('vprScoreRange'))
+        vpr_score['lte'] = upper_range_bound
+        vpr_score['gte'] = lower_range_bound
+    return vpr_score
+
+
+def request_uuid_export_vulnerabilities(args: Dict[str, Any]) -> PollResult:
+    """
+    Gets the UUID of the vulnerabilities export job.
+
+    Args:
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        PollResult: A result to return to the user which will be set as a CommandResults.
+    """
+    tag_category = args.get("tagCategory")
+    tag_value = args.get("tagValue")
+    request_params = remove_empty_elements(
+        {
+            'num_assets': arg_to_number(args.get('numAssets')),
+            'include_unlicensed': argToBoolean(args.get('includeUnlicensed')) if args.get('includeUnlicensed') else None,
+            'filters': {
+                'cidr_range': args.get('cidrRange'),
+                'first_found': relational_date_to_epoch_date_format(args.get('firstFound')),
+                'last_fixed': relational_date_to_epoch_date_format(args.get('lastFixed')),
+                'last_found': relational_date_to_epoch_date_format(args.get('lastFound')),
+                'network_id': args.get('networkId'),
+                'plugin_id': [arg_to_number(x) for x in argToList(args.get('pluginId'))],
+                'plugin_type': args.get('pluginType'),
+                'severity': argToList(args.get('severity')),
+                'since': relational_date_to_epoch_date_format(args.get('since')),
+                'state': argToList(args.get('state')),
+                'vpr_score': build_vpr_score(args),
+            }
+        }
+    )
+    if (tag_category and not(tag_value)) or (not(tag_category) and tag_value):
+        raise DemistoException('Please specify tagCategory and tagValue')
+    elif tag_category is not None and tag_value is not None:
+        if request_params.get('filters'):
+            request_params.get('filters')[f'tag.{tag_category}'] = tag_value
+        else:
+            request_params['filters'] = {f'tag.{tag_category}': tag_value}
```

Comment:
```suggestion
    if (tag_category and not tag_value) or (not tag_category and tag_value) or (not tag_category and not tag_value):
        raise DemistoException('Please specify tagCategory and tagValue')
    if request_params.get('filters'):
        request_params.get('filters')[f'tag.{tag_category}'] = tag_value
    else:
        request_params['filters'] = {f'tag.{tag_category}': tag_value}
```

Reply:
changed to another if because we can't send None values to the request body
```suggestion
    if tag_category and tag_value:
        if request_params.get('filters'):
            request_params.get('filters')[f'tag.{tag_category}'] = tag_value
        else:
            request_params['filters'] = {f'tag.{tag_category}': tag_value}
            
    if (tag_category and not(tag_value)) or (not(tag_category) and tag_value): 
        raise DemistoException('Please specify tagCategory and tagValue')
```